README.md

```diff
@@ -15,17 +15,17 @@ In the spirit of open source software development, jQuery always encourages comm
 3. [Writing Code for jQuery Foundation Projects](https://contribute.jquery.org/code/)
 
 
-Environments in which to use jQuery
+Environments which support jQuery
```

Comment:
This is not correct. Those environments are not doing anything specifically to support jQuery, it's jQuery that make sure it works fine in those environments.